x-pack/plugins/observability_solution/observability_logs_explorer/common/url_schema/url_schema_v1.ts

```diff
@@ -8,8 +8,12 @@
 import { availableControlsPanels, datasetSelectionPlainRT } from '@kbn/logs-explorer-plugin/common';
 import * as rt from 'io-ts';
 
-export const columnRT = rt.intersection([
+const allowedNamesRT = rt.union([rt.literal('content'), rt.literal('resource')]);
```

Comment:
**nit**: as suggested in the [io-ts documentation](https://github.com/gcanti/io-ts/blob/master/index.md#union-of-string-literals), prefer `keyof` to create literal unions:
```ts
const allowedNamesRT = rt.keyof({
  content: null,
  resource: null
})
```